Source/Core/DiscIO/WbfsBlob.cpp

```diff
@@ -40,10 +40,8 @@ WbfsFileReader::WbfsFileReader(const std::string& filename)
 
 WbfsFileReader::~WbfsFileReader()
 {
-	for (u32 i = 0; i != m_files.size(); ++ i)
-	{
-		delete m_files[i];
-	}
+	for (auto& f : m_files[i])
```

Comment:
But nobody really cares about what's being deleted imo.

I'll let you or @Buddybenj make the call.